server.js

```diff
@@ -4,33 +4,33 @@ const mongoose = require('mongoose');
 const bodyParser = require('body-parser');
 require('dotenv/config');
 
+// body-parser
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
+// Routes
 const tasksRoute = require('./routes/tasks');
-app.use('/api/tasks', tasksRoute);
+app.use('/api/tasks', tasksRoute); // create middleware
```

Comment:
I usually put comment above the code line, (not after)
```suggestion
// create middleware
app.use('/api/tasks', tasksRoute);
```